### ISSUE TITLE
Page went blank after im...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#919]
Notes
======
Page went blank after importing a project (overwrite)

Environment
======
Environment Key | Value
:--: | :--:
Version |11.0.0
Operating System | Darwin
Platform | darwin
Release | 15.6.0
Architecture | x64

Log History
======
```javascript
2016-12-01 12:42:26 I/ts-dashboard.html:1166:3: 
	Git Version 2.7.4 (Apple Git-66)
2016-12-01 12:42:26 I/migrator.js:31:2: Migrated projects
	[]
2016-12-01 16:58:36 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-01 16:59:19 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-01 16:59:44 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-01 17:0:25 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 10:18:1 I/ts-dashboard.html:1166:3: 
	Git Version 2.7.4 (Apple Git-66)
2016-12-02 10:18:1 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 10:28:56 I/ts-dashboard.html:1166:3: 
	Git Version 2.7.4 (Apple Git-66)
2016-12-02 10:28:57 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 10:29:49 I/ts-dashboard.html:1166:3: 
	Git Version 2.7.4 (Apple Git-66)
2016-12-02 10:29:49 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 11:5:10 I/ts-dashboard.html:1166:3: 
	Git Version 2.7.4 (Apple Git-66)
2016-12-02 11:5:10 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 11:11:3 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 11:11:6 I/ts-dashboard.html:385:28: Chunk Conflicts
	[
	  "01-03",
	  "01-07"
	]
2016-12-02 11:19:17 I/migrator.js:31:2: Migrated projects
	[
	  "en_php_text_ulb"
	]
2016-12-02 11:20:36 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]
2016-12-02 11:21:47 I/migrator.js:31:2: Migrated projects
	[
	  "en-x-demo1_php_text_ulb"
	]

```

**Comment by cckozie** [comment]
The app was not hung as I could go to settings and send feedback. Went back into 3-dot menu and clicked Home, and the home screen was presented with projects showing.
![image](https://cloud.githubusercontent.com/assets/8171759/20843709/0044a1c4-b882-11e6-946a-06c1d71131f8.png)
![image](https://cloud.githubusercontent.com/assets/8171759/20843720/0fb26c0e-b882-11e6-9f0b-75926c88ae9f.png)

**Comment by cckozie** [comment]
I have been able to replicate.
Load a project that has conflicts (or merge two to cause some)
Open the project and click on the triangle to display only the chunks that have conflicts
Go back Home and import with overwrite a project with the same attributes but without any conflicts
Open that imported project and you will see the blank screen until you click one of the mode icons

**Comment by cdwhitfield73** [comment]
This was fixed by implementing suggestion from issue #791.
